torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -571,8 +574,13 @@ def _fuse_conv_bn_qat_helper(m: GraphModule, is_cuda: bool) -> GraphModule:
 
     original_to_replacement_node = {}
     for r in replacements_with_conv_bias + replacements_no_conv_bias:
-        (replacement_conv_node, replacement_bn_node, replacement_getitem_node) =\
-            _get_conv_bn_getitem_nodes(r.replacements)
+        # TODO: rewrite this to return a map
+        (
+            replacement_conv_node,
+            replacement_bn_node,
+            replacement_getitem_node,
+            replacement_relu_node,
+        ) = _get_conv_bn_getitem_nodes(r.replacements)
```

Comment:
I guess this is for identifying the nodes, I feel https://github.com/pytorch/pytorch/pull/110743 should help